ADOPTERS.md

```diff
@@ -13,6 +13,8 @@ Below is a list of adopters of Antrea that have publicly shared the details
 of how they use it.
 
 **[Glasnostic](https://glasnostic.com)**
+Glasnostic makes modern cloud operations resilient. It does this by shaping how systems interact, automatically and in real-time. As a result, DevOps and SRE teams can deploy reliably, prevent failure and assure the customer experience. We use Antrea's Open vSwitch support to tune how services interact in Kubernetes clusters.
```

Comment:
@marcusschiesser would you mind wrapping the line at ~80 chars?

Reply:
sure not - I added the wrapping